client/service-worker.js

```diff
@@ -0,0 +1,41 @@
+// The Lounge - https://github.com/thelounge/lounge
+/* global clients */
+"use strict";
+
+self.addEventListener("push", function(event) {
```

Comment:
Webpacking a single file seems overkill to me. Also, Webpack generates fairly obfuscated code, and I don't know if we can/should ship sourcemaps for it, and we probably don't want to obfuscate what runs in service workers so people can see what they're running if they want (they can look at the repo, but that's different, IMO).

Anyway, I think we should save ourselves this trouble and just live it as is.